File: articles/automation-bias-and-human-overreliance/go/automation_bias_risk.go
package main

import "fmt"

func main() {
	acceptance := 0.93
	quality := 0.71
	uncertainty := 0.29
	reviewDeficit := 0.65
	overrideFriction := 0.72
	weakContestability := 0.0
	overrelianceGap := acceptance - quality
	if overrelianceGap < 0 {
		overrelianceGap = 0
	}
	score := (acceptance + overrelianceGap + uncertainty + reviewDeficit + overrideFriction + weakContestability) / 6.0
	fmt.Printf("automation_bias_risk_score=%.4f\n", score)
}

File: articles/automation-bias-and-human-overreliance/cpp/automation_bias_risk.cpp
#include <algorithm>
#include <iostream>

int main() {
    double acceptance = 0.93;
    double quality = 0.71;
    double uncertainty = 0.29;
    double review_deficit = 0.65;
    double override_friction = 0.72;
    double weak_contestability = 0.0;
    double overreliance_gap = std::max(0.0, acceptance - quality);
    double score = (acceptance + overreliance_gap + uncertainty + review_deficit + override_friction + weak_contestability) / 6.0;
    std::cout << "automation_bias_risk_score=" << score << "\n";
    return 0;
}
